### PR TITLE
github: Support docker image action

### PR DESCRIPTION
**Pull request by rzr**
Can be used for poorman CI/CL

### CODE REVIEW

**Comment by rzr** [comment]
@lanodan feel free to review and rebase your PR on it

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag my-image-name:$(date +%s)
```
Comment (lanodan) [inline .github/workflows/docker-image.yml:18]: That tag seems wrong, should probably be based on like a docker repository that Github might provide, nah?

cf. https://git.pleroma.social/pleroma/pleroma/-/blob/develop/ci/build_and_push.sh

**Comment by lanodan** [comment]
> @lanodan feel free to review and rebase your PR on it

The CppUnit PR? I think that one could be just based on a normal debian image workflow, at least a docker output wouldn't make much sense for it.

File: .github/workflows/docker-image.yml (hunk 1 above)
Comment (rzr) [reply .github/workflows/docker-image.yml]: feel free to improve

**Pull request merged**